src/DetectorConstruction.cxx

```diff
@@ -36,6 +36,10 @@
 
 using namespace std;
 
+int worldVol_x = 20*meter;
```

Comment:
Avoid globals. Put these in the header files as class fields, similar to `fIncB2eR`.